### ISSUE TITLE
enhet efter pris i butik

### ISSUE THREAD

**Issue by albinofsweden** [localfoodnodes/localfoodnodes#31]
Pris i butik måste anges baserat på enhet, inte innehåll i produkten..
Ex lammlåda - hel 169 kr/ 16 kg = fel
Ex lammlåda hel 169 kr/ kg = rätt

Det måste bara vara enheten, kg....annars blir det hel del

**Comment by davidajnered** [comment]
Då kommer det stå "lammlåda halv 84.5 kr / kg" (halva priset på en halv låda), och det blir ju inte rätt.

**Comment by davidajnered** [comment]
Possible fix https://github.com/localfoodnodes/localfoodnodes/commit/99449bba5d39ac32325bc8bdfc19843d0226a56b

**Comment by albinofsweden** [comment]
jo, det blir ju rätt om du ropar på rutan ”pris anges i” som ligger på Basic info….det är den infon ssom skall visas efter priset i checkout...
